Automattic/jetpack
# Updates Travis to generate code coverage reports

Changes phpunit to do code coverage and sends it to Codecov




#### Changes proposed in this Pull Request:

* This integrates codecov.io and Travis-CI.

#### Testing instructions:






* Check to see if coverage reports are generated by Travis.
* Check to see if Codecov gets reports.

#### Proposed changelog entry for your changes:

* N/A
